lib/auth/auth.go

```diff
@@ -3204,6 +3204,12 @@ func generateCert(ctx context.Context, a *Server, req certRequest, caType types.
 		return nil, trace.Wrap(err)
 	}
 
+	var githubUserID, githubUsername string
+	if githubIdentities := req.user.GetGithubIdentities(); len(githubIdentities) > 0 {
+		githubUserID = githubIdentities[0].UserID
+		githubUsername = githubIdentities[0].Username
+	}
+
```

Comment:
What is the impact of only respecting the first identity? Can it ever happen that user has multiple?

Reply:
i don't think there will be more than one. i will add a comment.